### PR TITLE
Create pages login and registration

### PR DESCRIPTION
**Pull request by Zenater**

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -40,10 +40,10 @@ body {
     height: 8px;
 }
 
-a {
-    color: inherit;
-    text-decoration: none;
-}
+/*a {*/
```
Comment (gitkoshelew) [inline src/styles/globals.css]: remove unused comments, please

File: src/pages/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <StylizedPaper title='Log in'>
+          <FormGroup className='text-sm'>
+            <span>Email</span>
+            <TextField type='email' className='mb-4' />
+            <span>Password</span>
+            <TextField type='password' className='mb-4' />
+            <div className='flex justify-between items-center'>
+              <Checkbox
+                size='small'
+                sx={{ color: '#2D3E6B', padding: '0px' }}
```
Comment (gitkoshelew) [inline src/pages/login/index.tsx]: why don't you use variables here? color shouldn't be hardcoded

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/styles/globals.css (hunk 1 above)
Comment (Zenater) [reply src/styles/globals.css]: i fexed

File: src/pages/login/index.tsx (hunk 2 above)
Comment (Zenater) [reply src/pages/login/index.tsx]: deleted this styles

**Review by gitkoshelew — APPROVED** [review]

**Review by visiteur85 — APPROVED** [review]

**Pull request merged**